src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const declarations = sourceString.split(';');
+
+  const resultingObject = {};
+
+  declarations.forEach((declaration) => {
+    if (declaration.trim() !== '') {
+      const [key, value] = declaration.split(':').map((item) => item.trim());
+
+      resultingObject[key] = value;
+    }
+  });
```

Comment:
Instead of using `forEach` and an if statement to filter out empty declarations, you can chain `filter` and `forEach` for better readability and to follow functional programming practices.